hudi-common/src/main/java/org/apache/hudi/common/table/TableSchemaResolver.java

```diff
@@ -389,7 +389,10 @@ private MessageType readSchemaFromLogFile(Path path) throws IOException {
    * @return
    */
   public static MessageType readSchemaFromLogFile(FileSystem fs, Path path) throws IOException {
-    try (Reader reader = HoodieLogFormat.newReader(fs, new HoodieLogFile(path), null)) {
+    // We only need to read the schema from the log block header,
```

Comment:
Can we make this method private and use the method above, which does not take FileSystem as an arg?

Reply:
This static method is also used elsewhere so that would be a larger cleanup.  The above private method relies on the member variable `metaClient` which is less flexible than this one.